doc/script_commands.txt

```diff
@@ -7263,12 +7267,13 @@ The function returns 1 if shop was updated successfully, or 0 if not found.
 
 NOTES:
  - That you cannot use -1 to specify default selling price!
```

Comment:
```suggestion
 - A sell price of -1 is not a valid value!
```

Reply:
We could actually implement that?
As suggested from @Toshiro90